src/test/lib/CMakeLists.txt

```diff
@@ -31,4 +31,6 @@ set_property(TARGET testlib PROPERTY FOLDER "tests")
 
 if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
     add_dependencies(platform MsQuicEtw)
+else()
+    target_compile_options(testlib PRIVATE ${QUIC_WARNING_FLAGS})
```

Comment:
This is actually already here because of the target_link_libraries warnings above. Did this actually do anything different?

Reply:
I checked, and if I take this out, I get the following compile error:
```
In file included from /vagrant/src/test/MsQuicTests.h:24:
/vagrant/src/inc/msquic.hpp:239:11: error: field 'Handle' will be initialized after field 'CloseAllConnectionsOnDelete' [-Werror,-Wreorder]
        : Handle(nullptr), CloseAllConnectionsOnDelete(false) {
          ^
/vagrant/src/inc/msquic.hpp:262:11: error: field 'Handle' will be initialized after field 'CloseAllConnectionsOnDelete' [-Werror,-Wreorder]
        : Handle(nullptr), CloseAllConnectionsOnDelete(false) {
          ^
/vagrant/src/inc/msquic.hpp:278:11: error: field 'Handle' will be initialized after field 'CloseAllConnectionsOnDelete' [-Werror,-Wreorder]
        : Handle(nullptr), CloseAllConnectionsOnDelete(false) {
          ^
```